tests/test_app_engine.py

```diff
@@ -41,6 +41,28 @@ def test_get_project_id_missing_apis():
     assert excinfo.match(r'App Engine APIs are not available')
 
 
+class TestSigner(object):
+    def test_key_id(self, app_identity_mock):
```

Comment:
Disabled for tests. This test file contains tests for two separate classes, so it leads that the test cases should be organized in classes.